src/Microsoft.Azure.WebJobs.Host/Bindings/BindingFactory.cs

```diff
@@ -176,7 +176,37 @@ public IBindingProvider BindToGenericItem<TAttribute>(Func<TAttribute, Type, Tas
             Func<TAttribute, ParameterInfo, INameResolver, Task<TAttribute>> postResolveHook = null)
             where TAttribute : Attribute
         {
-            var bindingProvider = new ExactTypeBindingProvider<TAttribute, TUserType>(_nameResolver, buildFromAttribute, buildParameterDescriptor, postResolveHook);
+            var bindingProvider = new ExactTypeBindingProvider<TAttribute, TUserType>(
+                _nameResolver, 
+                buildFromAttribute,
+                null,
+                buildParameterDescriptor, 
+                postResolveHook);
```

Comment:
defaults to null and is ignored. This is an advanced hook for legacy compat for the dashboard. 

---
In reply to: [95845742](https://github.com/Azure/azure-webjobs-sdk/pull/967#discussion_r95845742) [](ancestors = 95845742)